tests/unit/misc_fake.c

```diff
@@ -9,6 +9,7 @@
 #include <tilck/kernel/datetime.h>
 #include <tilck/kernel/sync.h>
 #include <tilck/kernel/sched.h>
+#include <tilck/kernel/fs/vfs_base.h>
```

Comment:
**FIXME:** Why including `vfs_base.h` here is necessary? There are no new changes in misc_fake.c that require it.